ui/app/authenticators/oidc.ts

```diff
@@ -0,0 +1,42 @@
+import OAuth2ImplicitGrantAuthenticator, {
+  parseResponse as ESAparseResponse,
+} from 'ember-simple-auth/authenticators/oauth2-implicit-grant';
+import { reject, resolve } from 'rsvp';
+
+import classic from 'ember-classic-decorator';
+
+interface SessionData {
+  token: string;
+}
+interface parseResponseObject {
+  authuser: string;
+  prompt: string;
+  scope: string;
+  code: string;
+  state: string;
+}
+
+@classic
+export default class OIDCAuthenticator extends OAuth2ImplicitGrantAuthenticator {
+  restore(data: SessionData): Promise<SessionData> {
+    if (data.token) {
+      window.localStorage.removeItem('waypointOIDCAuthMethod');
+      window.localStorage.removeItem('waypointOIDCNonce');
```

Comment:
Could we add a comment here to explain why we clean up these now?

Reply:
added, also lol this should have been in `authenticate` not `restore` 🤦